task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -153,8 +155,32 @@ def run(self, task_config: KubernetesTaskConfig) -> None:
                     ),
                 ),
             )
+        # TODO (TASKPROC-238): Add volume, cpu, gpu, desk, mem, etc.
+        container = V1Container(
+            image=task_config.image,
+            name=task_config.name,
+            command=[task_config.command]
```

Comment:
I was messing around with this branch so that i could more easily test something else and realized that we'll need to do one of two things: 
* use [shlex.split](https://docs.python.org/3.6/library/shlex.html#shlex.split) to get a list
* or, change KubernetesTaskConfig to have `command` be a PVector and have Tron pass us a list of `["cmd", "any", "desired", "args"]` (i.e., have Tron/our caller to the shlex.split())

otherwise, if we wanted to run something like `sleep 120` as a command, right now this code would try to run a command called `sleep 120` which won't exist:
```
    State:          Terminated
      Reason:       ContainerCannotRun
      Message:      OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"sleep 120\": executable file not found in $PATH": unknown```

Reply:
i think the closer to configuration step we do this, the better. best would be setting command to list[str] in paasta when generating tron configs.

Reply:
hmm, so looking closer at what people are doing in soaconfigs, it looks like there's several occurrences of constructs like `cmd && cmd2` - it turns out that in mesos we were relying on the fact that the containerizer would wrap the command in `/bin/sh -c` (https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L647)

I think we'll want to do something similar - have command be `["/bin/sh", "-c"]` and then we pass in the command from the KubernetesTaskConfig into `args` e.g.:
```Python
container = V1Container(
            image=task_config.image,
            name=task_config.name,
            # for compatibility with Mesos, we wrap all commands in a shell so that users can give us things like
            # cmd1 && cmd2
            command=["/bin/sh", "-c"],
            args=[task_config.command],
)
```

Reply:
(bonus of doing this change: we don't need to change anything elsewhere - although we probably want to have `setup_tron_namespace`/tron eventually pass us `/bin/sh -c`  as a command explicitly at some point in the future)